src/Tribe/Template/Month.php

```diff
@@ -179,6 +192,48 @@ public function __construct( $args = array() ) {
 		}
 
 		/**
+		 * Indicates if month view cache should be enabled or not.
+		 *
+		 * If the month view cache setting itself is not enabled (or not set) then this
+		 * method will always return false.
+		 *
+		 * In other cases, the default rules are to cache everything in the 2 months past
+		 * to 12 months in the future range. This policy can be refined or replaced via
+		 * the 'tribe_events_enable_month_view_cache' filter hook.
+		 *
+		 * @return bool
+		 */
+		protected function enable_month_view_cache() {
```

Comment:
Hmm, not quite down with that from the perspective that it may not be enabled. This method merely returns a recommendation which can then be overridden.

`should_enable_month_view_cache()` ?